### PR TITLE
Readme tweaks

### PR DESCRIPTION
**Pull request by ZevEisenberg**
Minor ReadMe cleanup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ Let's go back to our vanilla SwiftUI home view and enhance it using the Composab
 ```swift
 import ComposableNavigator
 
-struct HomeView: View {
+struct ExampleView: View {
```
Comment (ZevEisenberg) [inline README.md:154]: I thought this was right based on the comment below the code sample, but it's a little ambiguous. You should probably double check it.

Comment (ohitsdaniel) [reply README.md:154]: I would actually leave as HomeView. The example is picked up later again in the Navigation tree, where HomeView represents HomeScreen objects. :)

**Comment by ohitsdaniel** [comment]
Rest looks fine, thanks for the contribution! 🍨 🏅

**Comment by ohitsdaniel** [comment]
I just merged #68 and I think that should resolve the failing CI run. Could you rebase your PR?

**Review by ohitsdaniel — CHANGES_REQUESTED** [review]
Let's stick with HomeView instead of ExampleView. The rest looks great. 🙌

**Comment by ohitsdaniel** [comment]
Hey @ZevEisenberg ,

sorry for butchering your PR, Danger keeps failing when it's running from your fork and I can't really debug it unless the PR comes from an external fork. 😬 Will try my best to resolve this issue ASAP.

**Comment by ohitsdaniel** [comment]
Decided to temporarily disable Danger as it's not working with the standard GITHUB_TOKEN from forks. Will setup a bot account for Danger and use secrets in the future, but for now, I'll just disable it to unblock this PR. :)

**Comment by ZevEisenberg** [comment]
@ohitsdaniel should be all set

**Review by ohitsdaniel — APPROVED** [review]

**Pull request merged**